scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -511,26 +543,45 @@ class PropertySpec:
       non-alphanumeric characters to underscores.
 
     const:
-      The property's constant value as given in the binding, or None.
+      The property's constant value as given in the binding, or None. It can
+      be inherited from a matching pattern property.
 
     default:
       The property's default value as given in the binding, or None.
 
     deprecated:
-      True if the property is deprecated; False otherwise.
+      True if the property is deprecated; False otherwise. It can be inherited
+      from a matching pattern property. It can be inherited from a matching
+      pattern property.
```

Comment:
Duplicated sentence.
```suggestion
      True if the property is deprecated; False otherwise. It can be inherited
      from a matching pattern property.
```